src/sentry/incidents/models.py

```diff
@@ -282,13 +303,11 @@ class AlertRule(Model):
     # These will be removed after we've made these columns nullable. Moving to
     # QuerySubscription
     subscription_id = models.UUIDField(db_index=True, null=True)
-    dataset = models.TextField(null=True)
-    query = models.TextField(null=True)
-    aggregations = ArrayField(of=models.IntegerField, null=True)
-    time_window = models.IntegerField(null=True)
-    resolution = models.IntegerField(null=True)
+    aggregations = ArrayField(of=models.IntegerField)
+    query_subscription = FlexibleForeignKey("sentry.QuerySubscription", unique=True, null=True)
+    project = FlexibleForeignKey("sentry.Project", db_index=False, db_constraint=False, null=True)
```

Comment:
If an organization alert rule is applied to multiple projects will we have multiple records in the `sentry_alertrule` table with similar data?

Reply:
No, there will just be the one org level row, with multiple related subscriptions for each project.

Reply:
Are you converting this field to nullable to prepare it for future removal?

Reply:
Yeah, there's no need to have this field in the future since we'll have many projects related to each row.